lib/rucio/db/sqla/migrate_repo/versions/1c45d9730ca6_increase_identity_length.py

```diff
@@ -58,12 +58,12 @@ def upgrade():
         alter_column('account_map', 'identity', existing_type=sa.String(255), type_=sa.String(2048), nullable=False)
         create_foreign_key('ACCOUNT_MAP_ID_TYPE_FK', 'account_map', 'identities', ['identity', 'identity_type'], ['identity', 'identity_type'])
 
-        create_check_constraint(constraint_name='IDENTITIES_TYPE_CHK',
```

Comment:
Removed the constraint for mysql because with the latest version, mysql actually implements the constraints and without dropping the constraint, the creation fails because of duplicates. Unfortunately the drop syntax is different and alembic/sqlalchemy dont have the right syntax yet (see [https://github.com/sqlalchemy/alembic/issues/554](https://github.com/sqlalchemy/alembic/issues/554))